test/parallel/test-fs-readfilesync-pipe-large.js

```diff
@@ -26,12 +26,12 @@ const exec = require('child_process').exec;
 const f = JSON.stringify(__filename);
 const node = JSON.stringify(process.execPath);
 const cmd = `cat ${filename} | ${node} ${f} child`;
-exec(cmd, { maxBuffer: 1000000 }, function(err, stdout, stderr) {
+exec(cmd, { maxBuffer: 1000000 }, common.mustCall(function(err, stdout, stderr) {
```

Comment:
Hi, @sinoon , thanks for your contribution. We need to add new line here to make linter passed.

![Screen Shot 2019-04-29 at 12 16 40 AM](https://user-images.githubusercontent.com/23313266/56867131-5e919480-6a14-11e9-9cce-15fa5c689114.png)

Reply:
Thank you for remind.I fixed.